mscore/prefsdialog.ui

```diff
@@ -2499,28 +2499,8 @@
              </sizepolicy>
             </property>
             <property name="currentIndex">
-             <number>3</number>
+             <number>-1</number>
             </property>
-            <item>
-             <property name="text">
-              <string>Page Width</string>
-             </property>
-            </item>
-            <item>
-             <property name="text">
-              <string>Whole Page</string>
```

Comment:
For better look in QtDesigner, I'd rather leave these strings in but mark `notr="true"`, maybe with a remark ' translated elsewhere'